src/DIRAC/Core/Utilities/ReturnValues.py

```diff
@@ -5,14 +5,43 @@
 
    keys are converted to string
 """
+from __future__ import annotations
```

Comment:
It fixes a handful of annoying quirks when adding types. It was planned to be the default in 3.10 and then 3.11 but it has now been delayed twice due to having a different set of issues. [PEP-649](https://peps.python.org/pep-0649/) has some more details but for now I think `from __future__ import annotations` is the best course of action for DIRAC as it's more limited than the original approach. If we change our mind removing the future imports should be easier than the opposite approach.

Technically what it does it change the type hints to being evaluated on demand rather than at function definition time. For example:

```python
class C:
    @classmethod
    def factory_func(cls) -> C:
        return cls()
```

By default this fails with `NameError: name 'C' is not defined`. Alteratively you can make the type hint a string which causes it's evaluation to be delayed:

```python
class C:
    @classmethod
    def factory_func(cls) -> "C":
        return cls()
```

But this is quite ugly and annoying and makes evaluating the type hints harder. Using `from __future__ import annotations` makes it so that the type hint evaluation is always delayed. See [PEP-563](https://peps.python.org/pep-0563/ for details).

(Note this example isn't quite correct if you want correct typing for subclasses of `C`.)